SecurityPkg/Library/Tcg2InitEventLib/Tcg2InitEventLib.c

```diff
@@ -0,0 +1,148 @@
+/** @file
+  This instance of the Tcg2InitEventLib is available to initialize events
+  for logging purposes.

```

Comment:
There is no need to couple an assumption about how the library will be used.

```suggestion
  Initializes TCG event data structures.
```